proposals/proposal_1/test/unit_test.dart

```diff
@@ -0,0 +1,44 @@
+// This is an example unit test.
+//
+// Unit tests are handy for verifying the behavior of a single function, method,
+// or class. This file demonstrates how to test an example Controller class.
+//
+// Visit https://flutter.dev/docs/cookbook/testing/unit/introduction for
+// more information about Unit testing.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('MyController', () {
+    testWidgets('should start at 0', (WidgetTester tester) async {
+      // Arrange
+      final myController = MyController();
+
+      // Assert
+      expect(myController.count, 0);
+    });
+
+    testWidgets('should increment by 1', (WidgetTester tester) async {
+      // Arrange
+      final myController = MyController();
+
+      // Act
+      myController.increment();
+
+      // Assert
+      expect(myController.count, 1);
+    });
+  });
```

Comment:
Sounds good -- I'd argue for deleting the file entirely in that case in order to follow the "Can we build the functionality in a way that 90% of developers will want to keep it and/or add to it, instead of ripping it out?" rule.

Sound good?

Reply:
I do think that most devs will want to keep the file and add their tests to it. If they have to replace the `expect(1+1, notEqual(3));` line in the process, I think that's fine. WDYT?